Issue410 lesion color map integration

Gurmie12 opened the pull request:
Removed hard-coded color values in FreehandRoiTool to use color being passed down from DB through ROI of tool data for drawing new lesions and displaying older lesions.

Gurmie12 committed:
changed FreehandRoiTool to use color passed through roi from tool data

mkramers commented on src/stateManagement/toolColors.js:
```diff
@@ -27,8 +27,8 @@ function getActiveColor() {
 function getColorIfActive(data) {
-  if (data.color) {
-    return data.color;
+  if (data.roi.color) {
```
I don't think you intended/need this change. Please confirm and restore it.

Gurmie12 committed:
Fixed request changes.

Changed data.roi.color in getColorIfActive to allow for other tools (measurment and organs) to dispaly color and not crash on new object creation.

mkramers reviewed (changes requested):
@mkramers - to bump version before merging

mkramers committed:
Bump version number
